src/omv/boards/ARDUINO_GIGA/imlib_config.h

```diff
@@ -96,6 +96,12 @@
 
 // Enable find_apriltags() (64 KB)
 #define IMLIB_ENABLE_APRILTAGS
+//#define IMLIB_ENABLE_APRILTAGS_TAG16H5
+//#define IMLIB_ENABLE_APRILTAGS_TAG25H7
+//#define IMLIB_ENABLE_APRILTAGS_TAG25H9
+//#define IMLIB_ENABLE_APRILTAGS_TAG36H10
+#define IMLIB_ENABLE_APRILTAGS_TAG36H11
+//#define IMLIB_ENABLE_APRILTAGS_ARTOOLKIT
```

Comment:
```suggestion
#define IMLIB_ENABLE_APRILTAGS_ARTOOLKIT
```
I'd leave this one too by default for all boards, as it's popular.

Reply:
I've never seen people to use it. Sure? This isn't the same as the ArUco codes that OpenCV works with. I think it's just a old tag formt that AprilTags was using. It's #2 of this image: 
![image](https://github.com/user-attachments/assets/8b892a94-e9b9-4a6e-8e12-f1de787ebc5b)

Reply:
No I'm not sure, it just sounded familiar. Okay I guess the default one is fine.